src/actions/signupActions.js

```diff
@@ -0,0 +1,88 @@
+import axios from 'axios';
+import {
+  SIGNUP_ERROR,
+  SIGNUP_SUCCESS,
+  SIGNUP_STARTED,
+} from '../actionTypes/signupTypes';
+
+export /**
```

Comment:
I think it is better to have a specific export to actions instead of having it cover more than one function